### PR TITLE
Fixed minor typos

### PR DESCRIPTION
**Pull request by aadcg**
Minor documentation review.

### CODE REVIEW

**Comment by aadcg** [comment]
The latest commit is to fix the fact that when a user starts nyxt with the default cua-mode, he's greeted with `UNBOUND: Load URL`.

**Comment by aadcg** [comment]
Regarding the last commit, 10fbec6, the same could perhaps hold for other wm running on wayland but I haven't tested.

**Comment by aadcg** [comment]
On 10fbec6, I thought I had found a way to get bigger font sizes on sway. Though fonts indeed scale at startup, nyxt doesn't run very well with the GDK backend set to x11 on sway.

I don't understand why the scaling doesn't apply when the backend is wayland...

With that said, this commit should be ignored.

File: source/manual.lisp [hunk 1]
```diff
@@ -90,7 +90,7 @@ documentation.")
     (:p "See the " (:code "proxy-mode") " documentation.")
 
     (:h3 "Custom commands")
-    (:p "Creating your own invokable commands is similar to creating a Common
+    (:p "Creating your own invocable commands is similar to creating a Common
```
Comment (Ambrevar) [inline source/manual.lisp:93]: With a `k` is correct: https://en.wiktionary.org/wiki/invokable

File: source/base-mode.lisp [hunk 2]
```diff
@@ -19,7 +19,7 @@
                      "C-tab" 'switch-buffer-next
                      "C-pageup" 'switch-buffer-previous
                      "C-pagedown" 'switch-buffer-next
-                     "C-l" 'set-url-from-current-url
+                     "C-l" 'set-url
```
Comment (Ambrevar) [inline source/base-mode.lisp:22]: @jmercouris Wasn't this set to `set-url-from-current-url` on purpose?  I can't recall.

Comment (Ambrevar) [reply source/base-mode.lisp:22]: See 0bb2c1f46d517ca9ef080d0f881c13cbed8b7a7b

Comment (jmercouris) [reply source/base-mode.lisp:22]: Indeed it was the behavior that new users would expect.

File: source/manual.lisp (hunk 1 above)
Comment (aadcg) [reply source/manual.lisp:93]: Ok, fine for me

Comment (aadcg) [reply source/manual.lisp:93]: Is it possible to discard this commit from github?

File: source/base-mode.lisp (hunk 2 above)
Comment (aadcg) [reply source/base-mode.lisp:22]: Then I suggest changing mirroring these changes to emacs-mode and vim-mode and then show C-l as Load URL on the quickstart keys.

Comment (jmercouris) [reply source/base-mode.lisp:22]: That's probably a good idea. I'm just thinking about whether a more advanced user would want to have to C-k the whole input every time they set-url.

Comment (Ambrevar) [reply source/base-mode.lisp:22]: I find it more convenient to start with an empty input and press tab.

File: source/manual.lisp (hunk 1 above)
Comment (Ambrevar) [reply source/manual.lisp:93]: Yes, you can squash the commits locally, then force-push.
But I can also squash when merging on your behalf.

File: source/base-mode.lisp (hunk 2 above)
Comment (jmercouris) [reply source/base-mode.lisp:22]: Myself as well. Perhaps we can just bind set-url-from-current-url as well so it doesn't show up as undefined in Vi/Emacs mode.

Comment (Ambrevar) [reply source/base-mode.lisp:22]: To what should we bind set-url-from-current-url?

**Comment by Ambrevar** [comment]
Fixed the `even` typo in 188f74bd253f7e640d8b902cc581637b0ff958c5.
I've open #969 to continue the discussion on set-url vs. set-url-from-current-buffer.

**Pull request closed without merging**